### PR TITLE
Update ScriptDelegate.java

### PR DESCRIPTION
**Pull request by Dries1234**
Fixes # .

- [x] I have tested my changes.
- [ ] I don't have test my changes. I will do that in the following 7 days and aggre that this Pull request will be closed if I don't test it in the named time span.
- [x] I know the language good enough  for providing high quality code.

Changes proposed in this pull request:
---

-

-

### CODE REVIEW

File: src/main/java/ml/duncte123/skybot/objects/delegate/ScriptDelegate.java [hunk 1]
```diff
@@ -98,4 +98,18 @@ public boolean isEven(int number) {
...
+    String[] senArray = sen.split(" ");
+    for(int i = 0; i < senArray.length; ++i){
+
+        if(senArray[i].length() > longWordCount){
+            longWordCount = senArray[i].length();
+            longWord = senArray[i];
+        }
+    }
+    return longWord;
+}
```
Comment (duncte123) [inline src/main/java/ml/duncte123/skybot/objects/delegate/ScriptDelegate.java]: Fix your indentation and read the contributing guidelines please

Comment (Sanduhr32) [reply src/main/java/ml/duncte123/skybot/objects/delegate/ScriptDelegate.java]: The question is if we really need it? I mean i have 8h this weekend (from now to Sunday 10pm GTM+1) and would assign myself to add usefull methods to the ScriptDelegate.

Comment (Dries1234) [reply src/main/java/ml/duncte123/skybot/objects/delegate/ScriptDelegate.java]: Alright, you don't have to add it, this was really more as a random thing, at least now I know how to add pull requests :) and yes ill read them

Comment (duncte123) [reply src/main/java/ml/duncte123/skybot/objects/delegate/ScriptDelegate.java]: do we need this? nope

**Pull request closed without merging**